fmings/FEBourbonCollective
# MVP Styling My Collection & My Profile

## Description
Applied styling to My Collection and My Profile Pages


## Related Issue
#38 

## Motivation and Context
This change allows the user to experience an aesthetic and easy to interact with application.


## How Can This Be Tested?
Log in
Navigate through the my collection and my profile pages. Verify accuracy with wireframe.

## Screenshots (if appropriate):

## Types of changes
